be/src/vec/functions/function_utility.cpp

```diff
@@ -74,19 +74,12 @@ class FunctionSleep : public IFunction {
 
```

Comment:
Could you please add some comment to explain why we not using default_impl for const? 
"Sleep function should not be executed during open stage, this will makes fargment preparint too long, so we do not use default impl."